BugSplatDotNetStandard/BugSplat.cs

```diff
@@ -159,6 +221,14 @@ private void ThrowIfArgumentIsNullOrEmpty(string argument, string name)
             }
         }
 
+        private void ThrowIfHttpRequestFailed(HttpResponseMessage response)
+        {
+            if (!response.IsSuccessStatusCode)
+            {
+                throw new HttpRequestException(response.Content.ReadAsStringAsync().Result);
```

Comment:
This exception will print out the message received from the response. We can make this more generic if this is something that we feel shouldn't be exposed (for example, this could just be the generic API response message).

Reply:
LGTM!